app/frontend/src/cljs/teet/link/link_view.cljs

```diff
@@ -179,9 +179,10 @@
                                                             @selected-type
                                                             in-progress))))]
     [:div.links {:style {:margin "1rem 0"}}
+     (if (seq links)
```

Comment:
use `when` in cases where there is no else branch

Reply:
Done.